README.md

```diff
@@ -140,5 +140,35 @@ Added some limitations to avoid abuse.
 ### Contribution
 Any feedback, pull request or issue is welcome.
 
+### How to Contribute
+Fork this repo and then clone it:
+```
+git clone https://github.com/<your_name>/jsonbox.git
+```
+
+Cd into directory where the repo was cloned and install the dependencies:
+```
+npm install
+```
+
+You need MongoDB to run this application. It is recommended you install it yourself, for further info see the [official documentation](https://docs.mongodb.com/manual/installation/)
```

Comment:
You should mention there is also an easy option to follow the included instructions to create a local instance just by using `docker-compose` (see docker/docker.md)

Another easy option is to run a Mongo Docker container instead of installing it.
```
docker run -d -p 27017:27017 -v ~/data:/data/db mongo
```